packages/astro/src/core/build/generate.ts

```diff
@@ -64,6 +58,8 @@ import type {
 	StylesheetAsset,
 } from './types';
 import { getTimeStat } from './util.js';
+import { BuildPipeline } from './buildPipeline.js';
+import type { BufferEncoding } from 'vfile';
```

Comment:
nit: I think we can remove this import as `BufferEncoding` is a global with `@types/node` loaded.